simbad/util/pyrvapi_results.py

```diff
@@ -387,21 +388,13 @@ def create_lattice_results_tab(self, lattice_results, lattice_mr_results, result
             for i in range(0, results_to_display):
                 try:
                     pdb_code = df.loc[i][0]
-                    mr_program = list(df)[1][0:6]
-                    mr_workdir = os.path.join(
-                        self.work_dir, 'latt', 'mr_search', pdb_code, 'mr', mr_program)
-                    mr_log = os.path.join(
-                        mr_workdir, '{0}_mr.log'.format(pdb_code))
-                    ref_pdb = os.path.join(
-                        mr_workdir, 'refine', '{0}_refinement_output.pdb'.format(pdb_code))
-                    ref_mtz = os.path.join(
-                        mr_workdir, 'refine', '{0}_refinement_output.mtz'.format(pdb_code))
-                    ref_log = os.path.join(
-                        mr_workdir, 'refine', '{0}_ref.log'.format(pdb_code))
-                    ref_map = os.path.join(
-                        mr_workdir, 'refine', '{0}_refmac_2fofcwt.map'.format(pdb_code))
-                    diff_map = os.path.join(
-                        mr_workdir, 'refine', '{0}_refmac_fofcwt.map'.format(pdb_code))
+                    mr_workdir = os.path.join(self.work_dir, 'output_files')
```

Comment:
None of these should be inside the `try` clause. You are testing far too much ... the `os.path.join` will always work as long as both are defined ....

Take them outside the try and only test what you actually might fail.